src/schema/resolvers/user.js

```diff
@@ -8,52 +8,29 @@ export default {
     currentUser: (parent, args, context, info) => context.user
   },
   Mutation: {
-    createUser: async (parent, args, context, info) => {
-      return createUserDB(args.input, context)
-    },
     signup: async (parent, args, context, info) => {
 
-      let newUser = await context.models.user.findOne({ where: { username: args.data.username } })
-      let error = 'user already exists'
-      if (!newUser) {
-        newUser = await createUserDB(args.data, context)
-        error = null
-      }
+      let user = await context.models.user.findOne({ where: { username: args.data.username } })
+      if (user) return {authError: 'user exist' }
```

Comment:
There is a missing blank space between `{` and `authError`. Be sure to run the linter to be consistent with the syntax rules.